sdk/servicebus/service-bus/package.json

```diff
@@ -93,6 +93,9 @@
     "requiredResources": {
       "Azure Service Bus": "https://docs.microsoft.com/azure/service-bus-messaging"
     },
+    "dependencyOverrides": {
+      "@azure/identity": "^1.1.0"
```

Comment:
why not using `^1.3.0`?

Reply:
This is what the service-bus samples originally supported - ^1.1.0. Didn't want to change that, unless package owner wants to. ^1.1.0 means it will support all versions from 1.1.0 to the latest (1.3.0)

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/15632/files